sympy/stats/crv_types.py

```diff
@@ -1244,9 +1244,9 @@ def ExGaussian(name, mean, std, rate):
     ==========
 
     mu : A Real number, the mean of Gaussian component
-    std: A positive Real number,
+    std : A positive Real number,
```

Comment:
There seems to be some inconsistency between the function and the parameters described. While not caused by you, feel free to correct it. Function uses `name, mean, std, rate` so `std` is the only one actually matching the documentation.